src/main/java/teammates/ui/controller/AdminEmailImageUploadAction.java

```diff
@@ -43,15 +43,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
       
         data.isFileUploaded = true;
-        data.fileSrcUrl = Config.APP_URL + 
-                          Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
+        data.fileSrcUrl = Const.ActionURIs.PUBLIC_EMAIL_FILE_SERVE +
                           "?blob-key=" + 
                           blobKey.getKeyString();
         
-        log.info("New Image Uploaded : " + data.fileSrcUrl);
+        log.info("New Image Uploaded : " + Config.APP_URL + data.fileSrcUrl);
```

Comment:
I wonder if we should have a static method somewhere `getAbsoluteUrl(String)` and call that instead, making the code ever so slightly higher level. `Config.APP_URL +` is a lower level operation and I don't like that it is being repeated in many places.